commands-yml/commands/element/other/submit.yml

```diff
@@ -0,0 +1,79 @@
+---
+name: Submit Form
+short_description: Submit a FORM element
+description:
+  >
+     The submit command may also be applied to any element that is a descendant of a FORM element.
+
+     This feature is only supported from within WebView context.
```

Comment:
Need an article here: `within a WebView context`.

Reply:
Sorry. I meant a [grammatical article](https://en.wikipedia.org/wiki/Article_(grammar)) :)

Reply:
I'm working on re-working the context stuff and all the related docs (hybrid, web, iwdp, etc)